src/data/repositories/DataSetD2Repository.ts

```diff
@@ -40,11 +40,10 @@ export class DataSetD2Repository implements DataSetRepository {
     }
 
     private getDataSets(page: number, pageSize: number) {
-        return this.d2DataSetApi.get({
+        return this.d2DataSetApi.getWithOrgUnits({
             paging: { page, pageSize },
             filters: {},
             sorting: { field: "name", order: "asc" },
-            includeOrgUnits: true,
```

Comment:
`type D2DataSetOptions = GetDataSetOptions & { includeOrgUnits?: boolean };`

is not includeOrgUnits used?